eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -8,7 +8,7 @@
     "Agent": {
       "ubuntu-20.04": { "OSVmImage": "MMSUbuntu20.04", "Pool": "azsdk-pool-mms-ubuntu-2004-general" },
       "windows-2019": { "OSVmImage": "MMS2019", "Pool": "azsdk-pool-mms-win-2019-general" },
-      "macOS-10.15": { "OSVmImage": "macOS-10.15", "Pool": "Azure Pipelines" }
+      "macos-11": { "OSVmImage": "macos-11", "Pool": "Azure Pipelines" }
```

Comment:
I added this @sima-zhu . If you intentionally left it out I apologize!

Reply:
It is definitely needed. Not sure why I missed it.